pkg/pods/manifest.go

```diff
@@ -111,6 +113,9 @@ func (manifest *Manifest) Write(out io.Writer) error {
 	return nil
 }
 
+// Bytes() always returns the bytes from the original manifest file that was
```

Comment:
manifest.Bytes() is only used in tests, I modified the ones that depended on being able to programmatically change a manifest